core/state_processor_rip7560.go

```diff
@@ -516,25 +532,25 @@ func ApplyRip7560ExecutionPhase(
 	executionResult := CallFrame(st, &AA_ENTRY_POINT, sender, accountExecutionMsg, aatx.Gas)
 	receiptStatus := types.ReceiptStatusSuccessful
 	executionStatus := ExecutionStatusSuccess
+	execRefund := capRefund(st.state.GetRefund(), executionResult.UsedGas)
```

Comment:
I think it should be calculated and capped separately for each phase.
In in the future (see AA-450), refund it just after validation, before execution. This way, it might allow more TXs to be added to the block. Otherwise, this known-to-be-refunded gas can be used for more AATXs in the block.